### PR TITLE
styled discussion area and added comments

### LINKED ISSUE #338: Style Discussion Area and add date
## Description
The comments do not include the date they're created. The comments also seem to occupy more space. Style the Discussion Area and add the date to the comments. 

## Mocks
image

## Acceptance Criteria
Model Comment consists of a field date_created. https://github.com/systers/portal/blob/2db1ffc1854b5627423e1b76f35e0424249d078c/systers_portal/common/models.py#L29
Use this to include date for every comment and comments must be sorted by date.
Necessary changes have to be made in templates and views to get comments sorted by date. 
Discussion Area of Meetups
https://github.com/systers/portal/blob/2db1ffc1854b5627423e1b76f35e0424249d078c/systers_portal/templates/meetup/meetup.html#L45
Discussion Area of Support Requests
https://github.com/systers/portal/blob/2db1ffc1854b5627423e1b76f35e0424249d078c/systers_portal/templates/meetup/support_request.html#L40

### Update [Required]
- [ ] Include date for Meetup,Support Request comments
- [ ] Style Discussion Area of Meetups, Support Requests

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
1/2/3 hrs
First timer task

### PR DESCRIPTION
**Pull request by abhi20161997**
### Description
Styled discussion area and added date. Now the comments appear in an orderly manner with their corresponding dates of creation and are sorted from new to old.

Fixes #338 

###Mockups:
screenshot from 2018-03-19 19-29-01

### Type of Change:
Applied modifications to style.css file.
Added date field in the comments in both meetup.html and support_request.html

- Code
- User Interface

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?
Describe the tests you ran to verify your changes. Provide instructions or GIFs so we can reproduce. List any relevant details for your test.

### Checklist:
**Delete irrelevant options.**

- [] My PR follows the style guidelines of this project
- [] I have performed a self-review of my own code or materials
- [] My changes generate no new warnings 
- [] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by abhi20161997** [comment]
@sa7i , @tapasweni-pathak , @sammy1997 please review this.

**Comment by sa7i** [comment]
@abhi20161997 Could you post screenshots of your change?

**Comment by abhi20161997** [comment]
@sa7i  I have updated the screenshot in my PR. You can have a look at the mockups.

File: systers_portal/systers_portal/settings/dev.py [hunk 1]
```diff
@@ -6,9 +6,9 @@
...
     'default': {
         'ENGINE': 'django.db.backends.postgresql_psycopg2',
-        'NAME': 'systersdb',
-        'USER': '',
-        'PASSWORD': '',
+        'NAME': 'systerdb',
+        'USER': 'abhijit1',
+        'PASSWORD': 'pass1234',
         'HOST': 'localhost',
         'PORT': '5432',
```
Comment (sa7i) [inline systers_portal/systers_portal/settings/dev.py]: Do not commit this file.

**Comment by sa7i** [comment]
@abhi20161997  Changes to the css is affecting not only the discussion area but also others using box-containers class.
Sponsors Page looks like this.  
image
Font size of comment's author must match with the date. 
Add a dropdown for edit, delete options 
Date must be on the same line as author. e.g. github comments like " author commented on this date < dropdown for edit,delete>" or "author, date   < dropdown for edit, delete >" 
Comments are still occupying a lot of space, please try to minimize it.

**Comment by abhi20161997** [comment]
@sa7i I'll soon implement the mentioned changes.

**Comment by abhi20161997** [comment]
@sa7i shall I use edit and delete glyphicons instead of a dropdown ?

**Comment by sa7i** [comment]
@abhi20161997 you can use them.

**Comment by sammy1997** [comment]
I guess it's fine to use glyphicons as we are using bootstrap already. @sa7i However the bootstrap we are using is an older version I think as many of the glyphicons are not working. Should we create an issue to update the library? Views @abhi20161997 @sa7i ?

**Comment by sammy1997** [comment]
I guess it's fine to use glyphicons as we are using bootstrap already. @sa7i However the bootstrap we are using is an older version I think as many of the glyphicons are not working. Should we create an issue to update the library? Views @abhi20161997 @sa7i ?

**Comment by abhi20161997** [comment]
Ya @sammy1997  . I too wanted to update the bootstrap library as glyphicons like glyphicon glyphicon-option-vertical ,glyphicon glyphicon-menu-hamburger,glyphicon glyphicon-option-horizontal can't be used as of now. This may cause other problems in future too. @Janiceilene @sa7i  views ?

**Comment by abhi20161997** [comment]
@sa7i please review this.

**Comment by sa7i** [comment]
@abhi20161997 Thanks for the changes :) 
Edit glyphicon must be on the left side to the delete, not the other way round.  
Author must be in bold but not the date. 
@Jaemu What do you think of the colors? I feel colors aren't needed for the comments.

**Comment by sa7i** [comment]
@abhi20161997 Let's wait for @Jaemu to confirm about the colors. 
If colors aren't needed you may use the existing box-container class
Some more minor changes -
Only when clicked on "author" must it redirect to the author's profile. With the current changes, it's redirecting when clicked on whole "author commented on date" . Set href to only author.
Background-color isn't needed when hovered. Remove the edit,delete hover classes.

**Comment by abhi20161997** [comment]
@sa7i  cool :)

**Comment by abhi20161997** [comment]
@Janiceilene  your review on colours ?

**Comment by sa7i** [comment]
@abhi20161997 You can remove the color styling for the comments.

**Comment by abhi20161997** [comment]
@sa7i - I have removed the colors and implemented all changes you'd previously asked for. Please can you check this out.

**Comment by sa7i** [comment]
This looks good. Thanks for the work. :)

**Comment by sammy1997** [comment]
@mayburgos I can't seem to tag Vaibhavi on this. Her review request is blocking the merge!

**Comment by Vaibhavi24** [comment]
Hey @sammy1997 ! I'll be glad to review this if its fine with everyone :) @mayburgos

**Comment by sammy1997** [comment]
@Vaibhavi24 Please do! Your review was requested on this!

**Comment by sakshi1499** [comment]
@Vaibhavi24 Can you please review it? We cannot merge otherwise.

**Comment by sammy1997** [comment]
@sakshi1499 We can review it and merge this? @Vaibhavi24 has added her review

**Comment by sakshi1499** [comment]
@sammy1997 Shouldn't here be only one commit?

**Comment by sammy1997** [comment]
Yeah @sakshi1499 ! Sorry I didn't notice it... @abhi20161997 can you squash the 2 commits?

**Comment by sammy1997** [comment]
@abhi20161997? Are you working on it

**Comment by sakshi1499** [comment]
@abhi20161997 Can you please see this?

**Comment by sakshi1499** [comment]
@sammy1997 Should we close it then?

**Pull request merged**